modules/hitlnext/src/views/full/app/components/ConversationContainer.tsx

```diff
@@ -33,7 +33,7 @@ const ConversationContainer: FC<Props> = ({ api, bp }) => {
   async function handleResolve() {
     try {
       const handoff = await api.resolveHandoff(state.selectedHandoffId)
-      toast.success(lang.tr('module.hitlnext.handoff.resolved', { id: handoff.id }))
+      toast.success(lang.tr('module.hitlnext.handoff.resolved', { id: handoff.id }) , '' , {toasterProperties: {position: Position.TOP}})
```

Comment:
I'd suggest "toasterProps" instead.

Also, please install ESLint and Prettier as VSCode extensions. Normally it should auto-format your files when you save it. There should be a space between `{ toaster`

Reply:
Changes have been made as requested, I've intalled ESLint and prettier so the code should be formatted correctly.